auth/config.py.template

```diff
@@ -9,6 +9,8 @@ Sample configuration for the auth WSGI script.
 # For log levels
 import logging
 
+LOG_USING_NATIVE_PYTHON = True
+LOG_USING_SYSLOG = False
```

Comment:
I am sure this makes sense, but it looks counterintuitive to the noob: Aren't we using (r)syslog?
Maybe comment could help?